### PR TITLE
Add install instructions for CMake FetchContent

### LINKED ISSUE #156: Installation instruction could show how to use the project with CMake
It is possible to use the theme without having to clone it as a submodule by using `FetchContent` in CMake. 

In my CMake file, I have 

```cmake 
include(FetchContent)
fetchcontent_declare(
    doxygen-awesome-css
    URL https://github.com/jothepro/doxygen-awesome-css/archive/refs/heads/main.zip
)
fetchcontent_makeavailable(doxygen-awesome-css)

# Save the location the files were cloned into
fetchcontent_getproperties(doxygen-awesome-css SOURCE_DIR AWESOME_CSS_DIR)

# set input and output files
set(DOXYFILE_IN ${CMAKE_CURRENT_SOURCE_DIR}/doc/Doxyfile.in)
set(DOXYFILE_OUT ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile)
configure_file(${DOXYFILE_IN} ${DOXYFILE_OUT} @ONLY)
```

This downloads the latest main (but any other revision could be used) and unpacks in the build folder. The `Doxyfile.in` can reference this location in the `HTML_EXTRA_STYLESHEET` field 

```
HTML_EXTRA_STYLESHEET  = @AWESOME_CSS_DIR@/doxygen-awesome.css
```

When the configure stage of CMake is run, the `Doxyfile.in` is rendered to `Doxyfile` and Doxygen can be run as usual.

If this makes sense, I can submit a PR to add this to the installation instructions.

### PR DESCRIPTION
**Pull request by comkieffer**
Adds a section to README showing how the project can be installed with CMake. 

Closes #156

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +64,38 @@ cd doxygen-awesome-css
...
+# This allows us to get the path to doxygen-awesome.css
+fetchcontent_getproperties(doxygen-awesome-css SOURCE_DIR AWESOME_CSS_DIR)
+
+# Generate the Doxyfile
+set(DOXYFILE_IN ${CMAKE_CURRENT_SOURCE_DIR}/doc/Doxyfile.in)
+set(DOXYFILE_OUT ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile)
+configure_file(${DOXYFILE_IN} ${DOXYFILE_OUT} @ONLY)
+```
+
+This downloads the latest main (but any other revision could be used) and unpacks in the build folder. The `Doxyfile.in` can reference this location in the HTML_EXTRA_STYLESHEET field
```
Comment (jothepro) [inline README.md:91]:
```suggestion
This downloads the latest main (but any other revision could be used) and unpacks in the build folder. The `Doxyfile.in` can reference this location in the `HTML_EXTRA_STYLESHEET` field
```

File: README.md [hunk 2]
```diff
@@ -63,6 +64,38 @@ cd doxygen-awesome-css
...
+include(FetchContent)
+fetchcontent_declare(
+    doxygen-awesome-css
+    URL https://github.com/jothepro/doxygen-awesome-css/archive/refs/heads/main.zip
+)
+fetchcontent_makeavailable(doxygen-awesome-css)
+
+# Save the location the files were cloned into
+# This allows us to get the path to doxygen-awesome.css
+fetchcontent_getproperties(doxygen-awesome-css SOURCE_DIR AWESOME_CSS_DIR)
```
Comment (jothepro) [inline README.md:83]: I think I'd prefer the official capitalization or the CMake commands:

```suggestion
FetchContent_Declare(
    doxygen-awesome-css
    URL https://github.com/jothepro/doxygen-awesome-css/archive/refs/heads/main.zip
)
FetchContent_MakeAvailable(doxygen-awesome-css)

# Save the location the files were cloned into
# This allows us to get the path to doxygen-awesome.css
FetchContent_GetProperties(doxygen-awesome-css SOURCE_DIR AWESOME_CSS_DIR)
```

**Review by jothepro — COMMENTED** [review]
Thank you for the documentation! I have made some minor suggestions, I hope that you agree with them. :)

**Comment by comkieffer** [comment]
Sorry, I missed the notification for the changes. 

Your suggestions are a definite improvement.

Thanks for all your work.

LGTM!

**Pull request merged**